tests/ops/qubit/test_parametric_ops.py

```diff
@@ -2780,13 +2786,20 @@ def test_PauliRot_all_Identity_broadcasted(self):
         op = qml.PauliRot(theta, "II", wires=[0, 1])
         decomp_ops = op.decomposition()
 
-        phases = np.exp(-1j * theta / 2)
-        assert np.allclose(op.eigvals(), np.outer(phases, np.ones(4)))
-        mat = op.matrix()
-        for phase, sub_mat in zip(phases, mat):
-            assert np.allclose(sub_mat, phase * np.eye(4))
+        assert len(decomp_ops) == 1
+
+        decomp_op = decomp_ops[0]
+
+        assert decomp_op.name == "GlobalPhase"
```

Comment:
And again, probably easier to read and more comprehensive if we use a `assert qml.equal` here.  Though very soon we will also have `qml.assert_equal`!